src/static/js/modules/external-site-redirect.js

```diff
@@ -16,7 +16,7 @@ function init() {
     // Futher explanation can be viewed
     // at https://regex101.com/r/xT7sL5/2.
     var externalURLArray =
-        ( /(https?:\/\/(?:www\.)?(?!.*gov)(?!localhost).*)/g )
+        ( /(https?:\/\/(?:www\.)?(?!.*gov)(?!(?!content\.)localhost).*)/g )
```

Comment:
URL in https://github.com/cfpb/cfgov-refresh/pull/999/files#diff-5f71e0ec516ed4b791983ca746e435e8R17 needs update too.